sdk/keyvault/azure-security-keyvault-keys/test/ut/maasandbox.hpp

```diff
@@ -78,7 +78,7 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Keys { nam
 
     /** @brief Construct a new Attestation Client object from an existing attestation client.
      *
-     * @param attestationClient An existing attestation client.
+     * @param sandboxClient An existing sandbox client.
      */
     explicit MaaSandboxClient(MaaSandboxClient const& sandboxClient)
```

Comment:
I'm concerned about taking a dependency on MaaSandbox here, especially without first talking to the MAA team @anilba06 and @gkostal. The MaaSandbox is a test service that the MAA team uses, but it's not intended for public consumption.

Using it in private and then storing the quote that the service returned should be fine (that's what the attestation SDK does), but at a minimum, the MAA team should be told about this so they can understand the consequences.

The saved quote should be good for at least 6 months because the security baseline for SGX enclaves only resets every 6 months. And long-term, the attestation service is moving to a model where the client can determine the baseline which is used, so we can keep the quote longer once that happens.

Reply:
done, removed